### ISSUE TITLE
fs.mkdtempSync is not a function

### ISSUE THREAD

**Issue by mikeblakeuk** [karma-runner/karma-intellij#32]
Using WebStorm 2017.1
WebStorm 2017.1
Build #WS-171.3780.79, built on March 17, 2017
JRE: 1.8.0_112-release-736-b13 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 8.1 6.3

```
"C:\Program Files\nodejs\node.exe" "C:\Program Files (x86)\JetBrains\WebStorm 2017.1\plugins\js-karma\js_reporter\karma-intellij\lib\intellijServer.js" --karmaPackageDir=\node_modules\karma --configFile=\test\karma.webstorm.conf.js
intellij integration: failed to provide custom /debug.html: failed to create custom debug.html: 
[TypeError: fs.mkdtempSync is not a function]
Hash: f4xxxx
```

How can I tell what version of **fs** is being used?

**Comment by mikeblakeuk** [comment]
Node, v4.4.0 !

**Comment by segrey** [comment]
>intellij integration: failed to provide custom /debug.html

This warning indicates that the default `debug.html` will be served, so no real harm in it.

**Comment by segrey** [comment]
Closing as it's fixed in node 5.10
